source/reference/agent-api/index.md

```diff
@@ -2,163 +2,19 @@
 
 NodeX Agent is a resident Linux daemon process that provides NodeX Agent functionality as an HTTP-based API to another application, mainly within the Linux OS. Since this type of NodeX Agent provides HTTP-based API, it can be used regardless of the implementation language of the application that uses the API, and it can be used with zero learning cost by providing pre-built binaries. The pre-built binaries can be used with zero learning cost for building the API.
 
-## DID operations
+The following API is provided through the Unix Domain Socket (~/.nodex/run/nodex.sock) for applications using the NodeX Agent. that can hit the Unix Domain Socket to communicate with NodeX Agent through the socket.
 
-The following API is provided through the Unix Domain Socket (`~/.nodex/run/nodex.sock`) for applications using the NodeX Agent. that can hit the Unix Domain Socket to communicate with NodeX Agent through the socket.
-
-### Create DID
```

Comment:
I realized that my explanation was lacking. Please maintain the low-level API while separating it into a section for the high-level API. I would like to clarify that client applications should primarily use the high-level API.

Reply:
Classified into High-Level API and Low-Level API.

The composition of each API is as follows.
```
# High-Level API

## Data Operations
### transfer
### receive


# Low-Level API

## DID Operations
### Create DID
### Find DID

## Credential Operations
### Generate VC
### Verify VC
### Generate DIDComm Message (plaintext)
### Verify DIDComm Message (plaintext)
### Generate DIDComm Message (signed)
### Verify DIDComm Message (signed)
### Generate DIDComm Message (encrypted)
### Verify DIDComm Message (encrypted)
```

To help users understand which API to use, a description of each API has been added to the overview section.

Reply:
LGTM!